src/coreclr/jit/gentree.h

```diff
@@ -499,6 +499,8 @@ enum GenTreeFlags : unsigned int
     GTF_ADDRMODE_NO_CSE         = 0x80000000, // GT_ADD/GT_MUL/GT_LSH -- Do not CSE this node only, forms complex
                                               //                         addressing mode
 
+    GTF_CAST_STRESS_KEEP        = 0x00400000, // GT_CAST    -- A recommendation to keep this cast (used by stress mode)
```

Comment:
It should be in `GenTreeDebugFlags`.

Reply:
Suprisingly, CI + outerloops is fully green from without re-runs 😮